test/writer.xwiki

```diff
@@ -0,0 +1,862 @@
+*. Minus 3
+
+Minuses loose:
+
+
+*. Minus 1
+
+
+*. Minus 2
+
+
+*. Minus 3
+
+
+
+== Ordered ==
+
+Tight:
+
+
+1. First
+1. Second
+1. Third
+
+and:
+
+
+1. One
+1. Two
+1. Three
+
+Loose using tabs:
+
+
+1. First
+
+
+1. Second
+
+
+1. Third
+
+
+
+and using spaces:
+
+
+1. One
+
+
+1. Two
+
+
+1. Three
+
+
+
+Multiple paragraphs:
+
+
+1. Item 1, graf one.
+
+
+Item 1. graf two. The quick brown fox jumped over the lazy dog’s back.
+
+
+1. Item 2.
+
+
+1. Item 3.
+
+
+
+== Nested ==
+
+*. Tab
+**. Tab
+***. Tab
+
+Here’s another:
+
+
+1. First
+1. Second:
+1*. Fee
+1*. Fie
+1*. Foe
+1. Third
+
+Same thing but with paragraphs:
+
+
+1. First
+
+
+1. Second:
+
+
+1*. Fee
+1*. Fie
+1*. Foe
+1. Third
+
+
+
+== Tabs and spaces ==
+
+*. this is a list item indented with tabs
+
+
+*. this is a list item indented with spaces
+
+
+**. this is an example list item indented with tabs
+
+
+**. this is an example list item indented with spaces
+
+
+
+== Fancy list markers ==
+
+1. begins with 2
+1. and now 3
+
+
+with a continuation
+
+
+11. sublist with roman numerals, starting with 4
+11. more items
+111. a subsublist
+111. a subsublist
+
+Nesting:
+
+
+1. Upper Alpha
+11. Upper Roman.
+111. Decimal start with 6
+1111. Lower alpha with paren
+
+Autonumbering:
+
+
+1. Autonumber.
+1. More.
+11. Nested.
+
+Should not be a list item:
+
+
+M.A. 2007
+
+
+B. Williams
+
+
+
+----
+
+= Definition Lists =
+
+Tight using spaces:
+
+
+; apple
+: red fruit
+; orange
+: orange fruit
+; banana
+: yellow fruit
+
+Tight using tabs:
+
+
+; apple
+: red fruit
+; orange
+: orange fruit
+; banana
+: yellow fruit
+
+Loose:
+
+
+; apple
+: red fruit
+
+
+; orange
+: orange fruit
+
+
+; banana
+: yellow fruit
+
+
+
+Multiple blocks with italics:
+
+
+; //apple//
+: red fruit
+
+
+contains seeds, crisp, pleasant to taste
+
+
+; //orange//
+: orange fruit
+
+
+
+{{code}}
+{ orange code block }
+{{/code}}
+
+> orange block quote
+> 
+> 
+
+Multiple definitions, tight:
+
+
+; apple
+: red fruit
+: computer
+; orange
+: orange fruit
+: bank
+
+Multiple definitions, loose:
+
+
+; apple
+: red fruit
+
+
+: computer
+
+
+; orange
+: orange fruit
+
+
+: bank
+
+
+
+Blank line after term, indented marker, alternate markers:
+
+
+; apple
+: red fruit
+
+
+: computer
+
+
+; orange
+: orange fruit
+
+
+;1. sublist
+;1. sublist
+
+= HTML Blocks =
+
+Simple block on one line:
+
+
+foo
+And nested without indentation:
+
+
+foo
+
+
+bar
+Interpreted markdown in a table:
+
+
+
+
+
+This is //emphasized//
+
+
+And this is **strong**
+
+
+
+
+Here’s a simple block:
+
+
+foo
+
+
+This should be a code block, though:
+
+
+
+{{code}}
+<div>
+    foo
+</div>
+{{/code}}
+
+As should this:
+
+
+
+{{code}}
+<div>foo</div>
+{{/code}}
+
+Now, nested:
+
+
+foo
+This should just be an HTML comment:
+
+
+
+Multiline:
+
+
+
+
+Code block:
+
+
+
+{{code}}
+<!-- Comment -->
+{{/code}}
+
+Just plain comment, with trailing spaces on the line:
+
+
+
+Code:
+
+
+
+{{code}}
+<hr />
+{{/code}}
+
+Hr’s:
+
+
```

Comment:
Why all the blank lines here?

Reply:
These are RawBlocks containing HTML markup. I tried using the `html` macro, which works for some items, but for most items in testsuite.native I get:

```
Failed to execute the [html] macro. Cause: [When using the HTML macro inline, you can only use inline HTML content. Block HTML content (such as tables) cannot be displayed.
```

Previously I was reporting `BlockNotRendered`, which seems like it's probably the appropriate response.